metpy/plots/declarative.py

```diff
@@ -521,7 +521,11 @@ def draw(self):
                 panel.draw()
 
     def save(self, *args, **kwargs):
-        """Save the constructed graphic as an image file."""
+        """Save the constructed graphic as an image file.
+
+        This trait takes a string for saved file name. Additionally, the same arguments and
```

Comment:
Not a trait, but a method. Could also link to Matplotlib docs with `` `matplotlib.Figure.savefig` ``.

Reply:
Used `matplotlib.pyplot.savefig`.